src/third-parties/google-tag-manager/index.ts

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import data from './data.json';
+import data from '../../../data/google-tag-manager.json';
```

Comment:
Should we consider including absolute imports via alias (e.g. `@/data/...`)? Might be overkill and something we can worry about in a future PR